src/main/flight/autopilot.h

```diff
@@ -19,12 +19,19 @@
 
 #include "pg/autopilot.h"
 #include "flight/pid.h"
+#include "io/gps.h"
+
+extern float autopilotAngle[ANGLE_INDEX_COUNT]; // NOTE: ANGLES ARE IN CENTIDEGREES
```

Comment:
(ANGLE_INDEX_COUNT is 3 as in roll/pitch/yaw? It is more confusing than simply using `2`)

Reply:
Ha ha yeah I had to laugh also.  
I don't know where ANGLE_INDEX_COUNT = 2 came from.
It's gone now :-)